components/kyma-environment-broker/internal/broker/client.go

```diff
@@ -10,8 +10,8 @@ import (
 	"time"
 
 	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/pkg/errors"
```

Comment:
please don't use this archived package https://github.com/kyma-project/control-plane/pull/2234